gensim/models/word2vec.py

```diff
@@ -541,19 +552,19 @@ def scale_vocab(self, min_count=None, sample=None, dry_run=False):
             threshold_count = int(sample * (3 + sqrt(5)) / 2)
         downsample_total, downsample_unique = 0, 0
         for w in retain_words:
-            v = self.vocab[w]
-            word_probability = (sqrt(v.count / threshold_count) + 1) * (threshold_count / v.count)
+            v = self.raw_vocab[w]
+            word_probability = (sqrt(v / threshold_count) + 1) * (threshold_count / v)
```

Comment:
I've previously added "from __future__ import division" to the top for "true division"/Py3-style throughout.